cluster/charts/kustomization.yaml

```diff
@@ -0,0 +1,7 @@
+commonAnnotations:
+  helm.sh/hook: crd-install
+  helm.sh/hook-delete-policy: before-hook-creation
+
+resources:
```

Comment:
this pattern of using `kustomize` as part of the build process may be useful for adding other types of metadata/information to our generated CRDs that the controller-tools and kubebuilder markers don't support.  Also for dynamically filling in version information in a stack's app.yaml, install.yaml, etc. at build time when the version is known.